src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -12,7 +12,9 @@ define([
         interval: 30000,
         once: false,
         startatonce: true,
+        callEvent: "callMicroflow" | "callNanoflow",
         microflow: "",
+        nanoflow: null,
         firstIntervalAttr: null,
         intervalAttr: null,
         timerStatusAttr: null,
```

Comment:
post create add an error message when Timer event action does not have correct config (conditional required, microflow or nanoflow)

Reply:
Used mendix default `mx.ui.error("Error occured")`